main.css

```diff
@@ -0,0 +1,86 @@
+html, body {
+    height: 100%;
+    margin: 0;
+    font-family: 'Roboto Mono', monospace;
+}
+
+body {
+    display: grid;
+    grid-template:
+        "header" auto
+        "main" auto
+        "footer" auto
+        / 100vw;
+}
+
+.headerBar {
```

Comment:
Navigation bar is no longer sticky with new styles. I think sticky option was nicer, could you please revert those changes?

Reply:
Sticky navigation interrupts my homework assignment. I prefer not to have it sticky.

Reply:
Well, i am sure there is a solution on how to use sticky navigation with your assignment. If you wish i can look into that. Other than that i believe sticky one looks more professional.

Reply:
Simple I don't want sticky nav on my assignment. Nothing inherently professional about sticky headers. gets in the way for me